src/index.mjs

```diff
@@ -0,0 +1,220 @@
+// eslint-disable-next-line import/no-unresolved
+import tseslint from 'typescript-eslint';
+import globals from 'globals';
+import unicorn from 'eslint-plugin-unicorn';
+import sonarjs from 'eslint-plugin-sonarjs';
+import prettier from 'eslint-plugin-prettier/recommended';
+import jest from 'eslint-plugin-jest';
+import importPlugin from 'eslint-plugin-import';
+import js from '@eslint/js';
+
+export default [
+  js.configs.recommended,
+  ...tseslint.configs.recommended,
+  sonarjs.configs.recommended,
+  {
+    ignores: ['dist', 'coverage/*'],
+  },
+  {
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...globals.es2020,
+        ...globals.builtin,
+      },
+      parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: 'module',
+        ecmaFeatures: {
+          jsx: true,
+        },
+      },
+    },
+    plugins: {
+      import: importPlugin,
+      jest,
+      unicorn,
+    },
+    rules: {
+      'sonarjs/max-switch-cases': ['error', 15],
+      'sonarjs/no-empty-function': 'off',
+      'sonarjs/no-unused-expressions': 'off',
+      'sonarjs/todo-tag': 'off',
+      'sonarjs/fixme-tag': 'off',
+      'sonarjs/jsx-no-constructed-context-values': 'warn',
+      'sonarjs/no-array-index-key': 'warn',
+      'sonarjs/unused-import': 'off',
```

Comment:
Is this a duplicated rule?

Reply:
I turned it off because it was complaining about the same things as the `no-unused-vars`

Reply:
@braopa-smg

Reply:
`no-unused-vars` makes more sense to me here, as well. It catches the unused imports and unused stuff along the way. My vote is for keeping this as is.